src/tests/JIT/Regression/JitBlue/Runtime_34170/Runtime_34170.cs

```diff
@@ -21,7 +21,7 @@ public FloatNonAlignedFieldWithSmallOffset(float a)
 [StructLayout(LayoutKind.Explicit)]
 internal struct FloatNonAlignedFieldWithLargeOffset
 {
-    [FieldOffset(1021)]
+    [FieldOffset(0x10001)]
```

Comment:
This is just making the field offset large enough for the problem to happen even without stress.